src/beman/exemplar/identity.t.cpp

```diff
@@ -9,51 +9,43 @@
 
 namespace exe = beman::exemplar;
 
-TEST(IdentityTest, call_identity_with_int)
-{
-    for (int i = -100; i < 100; ++i)
-    {
+TEST(IdentityTest, call_identity_with_int) {
+    for (int i = -100; i < 100; ++i) {
         EXPECT_EQ(i, exe::identity()(i));
     }
 }
 
-TEST(IdentityTest, call_identity_with_custom_type)
-{
-    struct S
-    {
+TEST(IdentityTest, call_identity_with_custom_type) {
+    struct S {
         int i;
     };
 
-    for (int i = -100; i < 100; ++i)
-    {
+    for (int i = -100; i < 100; ++i) {
         const S s{i};
         const S s_id = exe::identity()(s);
         EXPECT_EQ(s.i, s_id.i);
     }
 }
 
-TEST(IdentityTest, compare_std_vs_beman)
-{
+TEST(IdentityTest, compare_std_vs_beman) {
 // Requires: std::identity support.
 #if defined(__cpp_lib_identity)
     std::identity std_id;
     exe::identity beman_id;
-    for (int i = -100; i < 100; ++i)
-    {
+    for (int i = -100; i < 100; ++i) {
         EXPECT_EQ(std_id(i), beman_id(i));
     }
 #endif
 }
 
-TEST(IdentityTest, check_is_transparent)
-{
+TEST(IdentityTest, check_is_transparent) {
 // Requires: transparent operators support.
 #if defined(__cpp_lib_transparent_operators)
 
     exe::identity id;
 
     const auto container = {1, 2, 3, 4, 5};
-    auto it = std::find(std::begin(container), std::end(container), 3);
+    auto       it        = std::find(std::begin(container), std::end(container), 3);
```

Comment:
That's a weird format rule - I suppose that's: AlignConsecutiveAssignments.  Anyway, I'd never write that -- maybe we can add a space after the container assignment so this is more normal spacing.

Reply:
Agreed!

Reply:
Do you mind re-posting this observation to #29 ? I personally find this janky as well. I expect we will update `.clang-format` after we fight it out in a (or many) weekly sync afterwards, this pr aims to provide a implementation as close as possible to that in the optional26 project.

Reply:
done